app/experimenter/experiments/constants/nimbus.py

```diff
@@ -421,6 +421,22 @@ class NimbusTargetingConfig:
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TARGETING_INFREQUENT_MAC_WIN_OR_LINUX_USER_WITH_5_BOOKMARKS = NimbusTargetingConfig(
+    name="Infrequent Mac, Win, or Linux user with 5 bookmarks",
+    slug="infrequent_mac_win_or_linux_user_with_5_bookmarks",
+    # 5 is the new profile bookmark count in Fx 100, a proxy for nothing imported
+    description="Between 1 and 6 days activity in the past 28 days, w/5 bookmarks",
+    targeting="({infrequent_user} && {mac_win_or_linux} && {fiveBookmarks})"
+    " || {sticky}".format(
+        infrequent_user=TARGETING_INFREQUENT_USER_URIS.targeting,
+        mac_win_or_linux="(os.isMac || os.isWindows || os.isLinux)",
```

Comment:
Why this way instead of omitting this?

Reply:
Since we didn't also include the various Tier >1 platorms (e.g. FreeBSD, NetBSD, etc, etc.).  Otherwise default queries will include them unless the query writer goes the to trouble of excluding them. That said, presumably these numbers would be small enough to get lost in the noise, but I figured I'd default to eliminating an extra variable, given that there's already plenty of complexity in data analysis.

I'm happy to do any of a) leave it alone, b) add a comment explaining the above, or c) simply drop that clause, whatever you'd prefer...

Reply:
If the concern is introducing an extra variable, it looks like this approach does so by being different from all the other targeting we've used.

Reply:
Heh, fair enough.  I'll remove it.